### DISCUSSION TITLE
V6 Beta Results

### DISCUSSION THREAD

**Discussion by DustinGrau** [gpstar81/GPStar-proton-pack#397] [General]
Provide feedback, good or bad, regarding the upgrade process for the ESP32 when evaluating the [V6.0.0-Beta](https://github.com/gpstar81/GPStar-proton-pack/releases/edit/V6.0.0-Beta) software. This is not for general issues with the pack or wand, only the instructions and process related to upgrading the ESP32 device.

**Comment by voodoo01x** [comment]
Install instructions were simple and easy, no issues with installing and back flashing back to 5.4.1
